platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheGracefulDegradationIntegrationTest.groovy

```diff
@@ -21,6 +21,25 @@ import org.gradle.api.internal.ConfigurationCacheDegradationController
 import javax.inject.Inject
 
 class ConfigurationCacheGracefulDegradationIntegrationTest extends AbstractConfigurationCacheIntegrationTest {
+    public static final String CONFIGURATION_CACHE_INCOMPATIBLE_TASKS_FOOTER = "Some tasks in this build are not compatible with the configuration cache."
+    public static final String CONFIGURATION_CACHE_DISABLED_REASON = "Configuration cache disabled because incompatible task"
+
+    def "a compatible build does not print degradation reasons"() {
+        buildFile """
+            System.getenv("HOME")  // Add a configuration input to force report to be generated
+            tasks.register("compatible") {
+                doLast {
+                    println("Hello")
+                }
+            }
+        """
+
+        when:
+        configurationCacheRun("compatible", "-D${LOG_REPORT_LINK_AS_WARNING}=true")
```

Comment:
❔  The issue mentions that the bug was observable with `--info` as well. Should we have a test scenario for that?